Src/ILGPU.Tests/CompareFloatOperations.tt

```diff
@@ -59,8 +59,10 @@ namespace ILGPU.Tests
 
 <# foreach (var (operationName, infix) in operationConfigurations) { #>
 <#      foreach (var type in FloatTypes) { #>
-<#         bool isFP16 = type == FloatTypes[0]; #>
-<#         var floatRanges = GetFloatRanges(type, true, true, true, false); #>
+<#         bool isFP8 = ( type == FloatTypes[0]) || (type == FloatTypes[1]); #>
```

Comment:
Question: should we allow comparison operations on these types? @MoFtZ @NullandKale 
I would argue that this may lead to unexpected performance characteristics. People should probably cast explicitly into a FP16 value to be aware of the performance considerations. What do you think?